### PR TITLE
Team page completed

### PR DESCRIPTION
**Pull request by pAditya198**
Screenshot (72)

### CODE REVIEW

**Comment by parth181195** [comment]
@pAditya198 can you make different component for profile page icon cause it will not open pop up the model

**Comment by parth181195** [comment]
we also need to make some changes to input config
like we need to pass team member details in config, it only needs to have name and role plus profile links

**Comment by pAditya198** [comment]
@parth181195 the profile page icon component that you are talking about is it similar to the speakerconfig or the speaker dialogue.

**Comment by pAditya198** [comment]
Screenshot (73)

@rossoskull @aashutoshrathi @parth181195  please review 
I've modified the Font and created a config for team as well as modified the team-card in the elements section for seperate data, and removed the pop-up in the team page.

**Comment by aashutoshrathi** [comment]
Can we atleast ensure this is Open Sans?

**Comment by pAditya198** [comment]
@aashutoshrathi yes I've ensured it online

File: src/app/components/team-page/team-page.component.html [hunk 1]
```diff
@@ -1 +1,47 @@
-<p>team-page works!</p>
+<div class="container">
+  <div id="speakers">
+    <div class='row center-lg center-xs center-md'>
+      <div class="col-xs-12">
+        <app-homepage-title title='Team' color='#419BF9'></app-homepage-title>
+      </div>
+      <p>DevFest Gandhinagar is brought to you with 💓 love,
```
Comment (aashutoshrathi) [inline src/app/components/team-page/team-page.component.html]: Can you replace this emoji with the simple :heart: emoji

Comment (pAditya198) [reply src/app/components/team-page/team-page.component.html]: I tried but couldn't because instead of red heart it displays white heart.

Comment (pAditya198) [reply src/app/components/team-page/team-page.component.html]: @aashutoshrathi modified as per required please review again.

**Comment by pAditya198** [comment]
Screenshot (75)

 Name and Post font weight changed and also added the red heart emoji

**Pull request merged**